_components/in-page-navigation/in-page-navigation.md

```diff
@@ -40,6 +40,9 @@ tags:
   - nav
 implementation:
   initProps:
+    - property: "`data-heading-selector`"
+      description: The list of header levels that should be included in the link list.
+      default: '"h2 h3"'
```

Comment:
I noticed there are backticks on `"h2 h3"` but it is not displaying as code on the build:

<img width="354" alt="image" src="https://github.com/uswds/uswds-site/assets/25211650/a4074fdb-ac6f-418c-b7cc-8c562d9b0ff0">

Reply:
Good catch! Updated in [5dcc4a1](https://github.com/uswds/uswds-site/pull/2252/commits/5dcc4a1f5b2007f48dad4536c43665f6354136c1)